.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
-node_modules/
-lib/
+node_modules
+flow-typed
```

Comment:
I'm not really sure if best practices for this is if we should requires committing the `flow-typed` directory or not. It seems like the definitions gets hashed for reproducible installs for everyone with the same versions.

I'm basing off `yarnpkg`, and `jest` which seem to both include this directory.